src/main/scala/com/lambtors/poker_api/module/poker/domain/model/Player.scala

```diff
@@ -0,0 +1,12 @@
+package com.lambtors.poker_api.module.poker.domain.model
+
+import java.util.UUID
+
+case class Player(playerId: PlayerId, gameId: GameId, firstCard: Card, secondCard: Card)
```

Comment:
I don't think it makes sense for a Player to have the gameId, because a player can be in more than one game at the same time (this happens a lot in online poker, for example)

Reply:
we have to talk about this... i think we can keep it as a requirement that a player should be only at one game at time, but we can talk about this.

if you want the game repository to have the players, whats the arquitecture you propose?